src/components/RecentForm/RecentForm.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./RecentForm.scss";
+
+const RecentForm = (props) => {
+  return (
+    <span className={`form ${props.data}`}>
+      {props.data.split("").map((game, i) => {
```

Comment:
BTW приучайся пропсы называть так, чтобы они имели смысл. Например
```suggestion
      {props.lastGames.split("").map((game, i) => {
```